src/configure

```diff
@@ -118,7 +118,7 @@ function rel2abs {
 }
 
 function read_value {
-  local val=`expr "X$1" : '[^=]*=\(.*\)'`;
```

Comment:
Can you explain the purpose of this change please?  I wish this read_value function were documented.  This `expr` thing is very obscure.

Reply:
Looking closely at this it appears the difference is $* would allow a multi-word value. 

For example:
x=`read_value foo bar`

In the previous code this would set x to "foo".

In the new code this would set x to "foo bar".

Why is this important?

We use this in read_dirname.  Imagine the case where your directory has a space:

CUDATKDIR=`read_dirname $1`

Imagine the CUDATKDIR sat at "/usr/local/cuda 10.1"

The previous code would set CUDATKDIR to /usr/local/cuda